app/models/file_set.rb

```diff
@@ -52,14 +45,10 @@ def ocr_file
     end
 
     def ocr_text
-      @ocr_text ||=
-        begin
-          if persisted? && File.exist?(ocr_file.gsub("file:", ""))
-            file = File.open(ocr_file.gsub("file:", ""))
-            ocr_doc = HOCRDocument.new(file)
-            ocr_doc.text.strip
-          end
-        end
+      return unless persisted? && File.exist?(ocr_file.gsub("file:", ""))
+      file = File.open(ocr_file.gsub("file:", ""))
```

Comment:
Do we need to remove some slashes in addition to `file:`, e.g., `file:///foo/bar` to `/foo/bar`?